README.md

```diff
@@ -58,6 +58,19 @@ Choose what you find more suitable for you:
 - [Spigen Rugged Armor Protective Case](https://www.amazon.com/Protective-Shock-Absorption-Anti-Scratch-Accessories-nintendo-switch/dp/B0B75N73N9/) - Suggested protective case made by Spigen.
 - [Maglass Screen Protector](https://www.amazon.com/Magglass-Tempered-Designed-Protector-Anti-Glare/dp/B09X82S4XL/) - Suggested screen protector made by Maglass.
 - [JSAUX Cooling Fans](https://jsaux.com/products/fan-cooler-for-steam-deck-gp0200) - Cooling fans for reducing overall temperatures made by JSAUX.
+- [NexDock](https://nexdock.com/features/) - Turn Steam Deck into a laptop.
+- [Killswitch](https://dbrand.com/killswitch) - Protective case made by dbrand for Steam Deck.
+- [Carrying Case](https://jsaux.com/products/upgraded-carrying-case-for-steam-deck-bg0106) - Protective case for Steam Deck for accessories, chargers and cables.
+- [Standing Case](https://jsaux.com/products/upgraded-protective-standing-case-for-steam-deck-pc0105) - Protective Standing Case for Steam Deck.
+- [Silicon Case](https://jsaux.com/products/steam-deck-protective-case) - Silicon Protective Case for Steam Deck.
+- [6-in-1 Docking Station](https://jsaux.com/products/upgraded-docking-station-for-steam-deck) - JSAUX 6-in-1 Docking Station for Steam Deck.
+- [Steam Deck Dock](https://store.steampowered.com/steamdeckdock) - Steam Deck Docking Station by Valve.
+- [iVoler Dock](https://www.ivoler.com/products/upgraded-docking-station-for-steam-deck) - iVoler 6in-1 Docking Station for Steam Deck.
```

Comment:
Unfortunately this link is not working for me...it redirects back to it-it page.  
Is the same for you?

Reply:
I've just tried in incognito and with another browser but points to the right product in En [screen here](https://i.imgur.com/ewxsp7j.jpg). Could you please try it again?

Reply:
I've tried with another device and different browsers, same result.
I think it's something related to my ISP, so I'll mark this as legitimate!